index.ios.js

```diff
@@ -13,19 +13,25 @@ import React, {
 import { Provider } from 'react-redux';
 import { createStore, applyMiddleware } from 'redux';
 import rootReducer from './app/reducers/rootReducer.js';
-import Map from './app/components/Map.js';
+import ViewContainer from './app/components/ViewContainer.js';
+import Menu from './app/components/Menu.js';
+
+const SideMenu = require('react-native-side-menu');
+
 var store = createStore(rootReducer);
 
 class findAR extends Component {
   render() {
+    const menu = <Menu onItemSelected={this.onMenuItemSelected} navigator={navigator}/>;
```

Comment:
Got error:
Warning: Failed propType: Required prop `onItemSelected` was not specified in `Menu`. Check the render method of `findAR`.